index.html

```diff
@@ -155,6 +155,15 @@ <h2>
         for payment. Concretely, a payment handler defines:
       </p>
       <dl>
+        <dt>
+          <dfn>Steps to convert and validate payment method data</dfn>
+        </dt>
+        <dd>
+          How a <a>payment method</a> [=converted to an IDL value|converts=] and
```

Comment:
Hmm... I think conversion and validation need to be separate things. Conversion doesn't need its own steps: it's clearly defined by the WebIDL step (i.e., it's redundant and shouldn't be part of this). Validation, on the other hand should be specified.

Reply:
I'm sympathetic to that argument. I wrote it like this for two reasons:

1. It was awkward to find the words to say "if a spec says to convert it, then do, and also if a spec has `steps to validate the converted data`, do those too."
2. For SPC, I was actually hoping to normalize the data as part of 'conversion' (see what is currently step 5 of https://w3c.github.io/secure-payment-confirmation/#sctn-steps-to-check-if-a-payment-can-be-made), which means that it takes on a meaning beyond the basic WebIDL meaning.

I'm happy to poke at the wording here myself, but interested in your thoughts on #2 :). (A reasonable counter-argument is to point out that we're likely to move away from the PR API for SPC anyway, and so it shouldn't be bent to make SPC work better!)

Reply:
Hmm...  yeah, that algorithm (2) seems to overstep a bit... it's doing checking, but it's also mutating an object. Editorially, I'd recommend splitting that (personally, that's come back to bite me quite a few times). 

Nevertheless, I get what you are trying to do now... but I'd still urge your to split those into two if possible.